test/quantization/test_quantize_script.py

```diff
@@ -2574,3 +2600,231 @@ def forward(self, x):
+                    'fc3': default_qconfig
+                }
+                model_traced = torch.jit.trace(script_model, self.calib_data[0][0])
+                model_script = torch.jit.script(script_model)
+                result_eager = model_eager(self.calib_data[0][0])
+                for model_under_test in [model_traced, model_script]:
+                    model_quantized = quantize_script(
+                        model_under_test,
+                        qconfig_dict,
+                        test_only_eval_fn,
+                        [self.calib_data],
+                        inplace=False)
+                    self.assertEqual(model_quantized(self.calib_data[0][0]), result_eager)
+
+    def test_skip_quant(self):
+        """ Test None qconfig
+        """
+        for qengine in supported_qengines:
+            with override_quantized_engine(qengine):
+                # Eager mode
+                eager_model = AnnotatedSkipQuantModel(qengine).eval()
+
+                # Graph mode
+                script_model = SkipQuantModel().eval()
+                # Copy weights for eager_model
+                script_model.sub.fc1.weight = torch.nn.Parameter(eager_model.sub.module.fc1.weight.detach())
+                script_model.sub.fc1.bias = torch.nn.Parameter(eager_model.sub.module.fc1.bias.detach())
+                script_model.sub.fc2.weight = torch.nn.Parameter(eager_model.sub.module.fc2.weight.detach())
+                script_model.sub.fc2.bias = torch.nn.Parameter(eager_model.sub.module.fc2.bias.detach())
+                script_model.fc.weight = torch.nn.Parameter(eager_model.fc.weight.detach())
+                script_model.fc.bias = torch.nn.Parameter(eager_model.fc.bias.detach())
+
+                model_eager = quantize(eager_model, test_only_eval_fn, self.calib_data)
+                qconfig_dict = {
+                    '': torch.quantization.get_default_qconfig(qengine),
+                    'fc': None
+                }
+                model_traced = torch.jit.trace(script_model, self.calib_data[0][0])
+                model_script = torch.jit.script(script_model)
+                result_eager = model_eager(self.calib_data[0][0])
+                for model_under_test in [model_traced, model_script]:
+                    model_quantized = quantize_script(
+                        model_under_test,
+                        qconfig_dict,
+                        test_only_eval_fn,
+                        [self.calib_data],
+                        inplace=False)
+                    self.assertEqual(model_quantized(self.calib_data[0][0]), result_eager)
+
+    def test_single_linear_dynamic(self):
+        r"""Compare the result of dynamic quantization of single linear layer in
+        eager mode and graph mode.
+        """
+        if 'qnnpack' in supported_qengines:
+            with override_quantized_engine('qnnpack'):
```

Comment:
you can use decorator and then check if the current engine is qnnpack, you can add a similar function: https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_quantized.py#L127